src/probnum/random_variables/_normal.py

```diff
@@ -8,14 +8,8 @@
 
 from probnum import utils as _utils
 from probnum.linalg import linops
-from probnum.type import (
-    ShapeType,
-    # Argument Types
-    ArrayLikeGetitemArgType,
-    FloatArgType,
-    RandomStateArgType,
-    ShapeArgType,
-)
+from probnum.type import ArrayLikeGetitemArgType  # Argument Types
+from probnum.type import FloatArgType, RandomStateArgType, ShapeArgType, ShapeType
```

Comment:
Same as in the previous comment. Just replicate whatever you end up doing in `src/probnum/random_variables/_dirac.py`.